frontend/src/app/modules/posts/components/add-comment/add-comment.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="card w-96 w-[480px] flex flex-nowrap">
+  <div class="flex items-center">
+    <div class="flex-none mt-1">
+      <div class="avatar">
+        <div class="w-10 rounded-full">
+          <img src="https://daisyui.com/images/stock/photo-1534528741775-53994a69daeb.jpg" />
+        </div>
+      </div>
+    </div>
+    <form [formGroup]="myForm">
+      <div [ngClass]="{'w-[480px]': !isLoading, 'w-[430px]': isLoading}" class="relative px-2 ml-2 flex flex-nowrap">
+        <textarea id="content" formControlName="content" class="text-black w-full" placeholder="Add a comment..." name="content" type="text"></textarea>
+        <button *ngIf="!isLoading" class="material-symbols-outlined cursor-pointer absolute top-0 right-8 ">send</button>
```

Comment:
Co myślisz o tym żeby dać tutaj `type="submit"`, dzięki czemu wystarczyłoby wcisnąć enter żeby dodać komentarz zamiast klikać myszką?

Reply:
nie do konca to dziala, uzylem       (keydown.enter)="onFormSubmit()"